src/ansible_navigator/actions/doc.py

```diff
@@ -182,8 +182,9 @@ def _run_runner(self) -> Union[None, dict, Tuple[str, str, int]]:
                 [self._plugin_name], plugin_type=self._plugin_type, playbook_dir=playbook_dir
             )
             if plugin_doc_err:
-                msg = "Error occurred while fetching doc for" " plugin {0}: '{1}'".format(
-                    self._plugin_name, plugin_doc_err
+                msg = (
+                    "Error occurred while fetching doc for plugin"
+                    f" {self._plugin_name}: '{plugin_doc_err}'"
```

Comment:
I see that you pass this to the logger. For loggers, the rules are different — they interpolate strings internally, and it's lazy, meaning that it may even be skipped based on the logging settings.
The proper way is to pass the template string as the first argument, and pass the values as positional arguments.

P.S. Another reason to do this is that some tools that collect logging statistics are better at counting which invocation was hit multiple times based on the static template rather than different pre-rendered strings passed to the same logging call.

Reply:
yeah it definitely going to the logger, easy enough to move, not that it was computationally expensive :)

Reply:
Right, I guess pylint missed this because it's a separate var that it didn't understand that it'll end up being passed to the logger.